packages/venia-ui/lib/components/Field/message.js

```diff
@@ -1,18 +1,31 @@
 import React from 'react';
+import { useIntl } from 'react-intl';
 import { node, shape, string } from 'prop-types';
 
 import defaultClasses from './message.css';
 import { mergeClasses } from '../../classify';
 
 const Message = props => {
     const { children, classes: propClasses, fieldState } = props;
+    const { formatMessage } = useIntl();
     const { asyncError, error } = fieldState;
     const errorMessage = error || asyncError;
 
     const classes = mergeClasses(defaultClasses, propClasses);
     const className = errorMessage ? classes.root_error : classes.root;
+    let translatedErrorMessage;
 
-    return <p className={className}>{errorMessage || children}</p>;
+    if (errorMessage) {
```

Comment:
As written, this assumes `asyncError` is the same format, but I think @jimbo said that the fieldState no longer returns an `asyncError` prop.

Can you validate that? If it no longer has the prop, you can remove the supporting code here for `asyncError` and then your current conditional here works. However, If we have to continue supporting `asyncError` we should just use `if(error)` here and use its props within this conditional.